Source/Renderer/AutomaticUniforms.js

```diff
@@ -1560,6 +1560,34 @@ define([
             getValue : function(uniformState) {
                 return uniformState.shadowMap.lightDirectionEC;
             }
+        }),
+
+        /**
+         * An automatic GLSL uniform representing the sun's shadow map light position in eye coordinates.
+         *
+         * @alias czm_sunShadowMapLightPositionEC
+         * @glslUniform
+         */
+        czm_sunShadowMapLightPositionEC : new AutomaticUniform({
+            size : 1,
+            datatype : WebGLConstants.FLOAT_VEC3,
+            getValue : function(uniformState) {
+                return uniformState.shadowMap.lightPositionEC;
+            }
+        }),
+
+        /**
+         * An automatic GLSL uniform representing the sun's shadow map radius.
+         *
+         * @alias czm_sunShadowMapRadius
+         * @glslUniform
+         */
+        czm_sunShadowMapRadius : new AutomaticUniform({
```

Comment:
Combine this with the above uniform to make a `vec4`?

Also, what is the radius?  Attenuation?